extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/theta/SketchBufferAggregator.java

```diff
@@ -40,6 +44,7 @@
   private final int maxIntermediateSize;
   private final IdentityHashMap<ByteBuffer, Int2ObjectMap<Union>> unions = new IdentityHashMap<>();
```

Comment:
Please annotate `@GuardedBy("stripedLock")`

Reply:
Please also add all the necessary `@GuardedBy("stripedLock")` annotations to fields and methods in other Aggregator classes changed in this PR.